signup/signup.php

```diff
@@ -81,12 +71,12 @@
 			<h3 class="form-subheading">Background Information</h3>
 
 			<ul class="dcf-pl-0">
-				<!-- TODO: NOTE THAT THIS WAS COMMENTED OUT, WE CURRENTLY DONT LET THEM SAY THE LANGUAGE THEY REQUIRED, WE LET IT DEFAULT TO ENGLISH
+				<!-- TODO: NOTE THAT THIS WAS COMMENTED OUT, WE CURRENTLY DONT LET THEM SAY THE LANGUAGE THEY REQUIRED, WE LET IT DEFAULT TO ENGLISH -->
 				<li class="form-radio" id="language">
 					<label for="language" class="form-required">Which language will you require?</label>
```

Comment:
Maybe workshop this text--could consider "Which language would you prefer?" or something. Maybe ask Tracy/Dr Moody on it

Reply:
I switched the wording and Tracy was good with it. Good call!